app/src/main/java/io/euphoria/xkcd/app/impl/ui/views/InputBarView.java

```diff
@@ -169,7 +170,7 @@ private void doNickChange() {
         // Strip whitespace
         Editable nickEditor = nickEntry.getText();
         String newNick = nickEditor.toString();
-        String trimmedNick = newNick.trim();
+        String trimmedNick = trimUnicodeWhitespace(newNick);
```

Comment:
I also used this method of trimming Strings in a unicode aware way for the InputBar nick entry field. Is this desirable?

Reply:
It might be desirable to follow what the upstream frontend does.

Both judging by the source code and experimentally, the it does in fact *not* trim whitespace, letting the backend sanitize it out and receiving the trimmed via the `nick-reply`.

If we *do* trim whitespace, we should definitely not use the arguably broken Java method (okay, it trims C0 controls, but why not C1 ones, and why does it trim non-whitespace in the first place?), but `trimUnicodeWhitespace()` instead.

Reply:
resolved by 30504c31d88cfc803d1a3621715c62ab2fa7d187